litmus-portal/namespaced-K8s-template.yml

```diff
@@ -153,9 +153,7 @@ spec:
                   name: litmus-portal-admin-config
                   key: JWTSecret
             - name: LITMUS_PORTAL_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
+              value: ${LITMUS_PORTAL_NAMESPACE}
```

Comment:
How will this value be parsed?

Reply:
It is being taken as user input all over the manifest.. @gdsoumya cc: @rajdas98

Reply:
it is taken as user input  all over the manifest @gdsoumya cc: @rajdas98

Reply:
Did not get you? How is this value being recieved? This manifest will be applied directly by the user using `kubectl apply` right?